src/libs/ajax.js

```diff
@@ -1056,6 +1056,10 @@ const Runtimes = signal => ({
     return res.json()
   },
 
+  invalidateCookie: () => {
+    return fetchLeo(`proxy/invalidateToken`, _.merge(authOpts(), { signal }))
```

Comment:
```suggestion
    return fetchLeo('proxy/invalidateToken', _.merge(authOpts(), { signal }))
```

Reply:
Going to push back against this one. The backticks align with the setCookie endpoint above. I am happy to change both of them at once, but wanted to maintain consistency within the single Ajax object.